django/common/adapters/fhir_api.py

```diff
@@ -30,6 +30,16 @@ def create(self, resource_type: str, payload: dict, auth_token=None):
             self._db[resource_type] = [resource]
         return resource
 
+    def validate(self, resource_type: str, payload: dict, auth_token=None):
```

Comment:
Do we want to unit test this?

Reply:
it is unit tested

Reply:
done

Reply:
Finally, I don't think we want to if this method simply returns a hard coded dict
It could be useful if we had a real validation mechanism though